modules/custom/cu_photo_gallery_bundle/cu_photo_gallery_bundle.module

```diff
@@ -38,7 +38,12 @@ function cu_photo_gallery_bundle_secure_permissions($role) {
       'edit any photo_gallery content',
       'edit own photo_gallery content',
     ),
-    'edit_my_content' => array(),
+    'edit_my_content' => array(
```

Comment:
Is there a "view photo_gallery content" permission and does that need to be included.

Reply:
also is there a "delete own" permission?

Reply:
Edit my content and edit only can't delete their own content according to the node permissions on our 3.0 user roles doc. All permissions are from node.